completion_aggregator/serializers.py

```diff
@@ -255,6 +256,19 @@ def get_username(self, obj):
         """
         return obj.user.username
 
+    def get_mean(self, obj):
+        """
+        Return the mean completion percent for all enrolled users.
+        """
+        enrollments = compat.get_users_enrolled_in(obj.course_key)
+        enrollment_count = enrollments.count()
+
+        mean = Aggregator.objects.filter(
+            course_key=obj.course_key,
+            aggregation_name='course',
+        ).aggregate(Sum('percent')).get('percent__sum', 0) / enrollment_count
```

Comment:
We should probably check first to make sure `enrollment_count` is not zero, even though that should be rare.

Reply:
Done, I'm returning 0 in that case.